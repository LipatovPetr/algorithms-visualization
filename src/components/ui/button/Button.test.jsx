import { render, screen } from "@testing-library/react";
import { Button } from "./Button.tsx";

test("Render Button with text", () => {
  render(<Button text="Текст кнопки" />);
  const button = screen.getByText("Текст кнопки");
  expect(button).toBeInTheDocument();
  expect(button).toMatchSnapshot();
});
