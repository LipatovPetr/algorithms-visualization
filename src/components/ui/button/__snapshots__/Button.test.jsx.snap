// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Button with text 1`] = `
<p
  class="text undefined"
>
  Текст кнопки
</p>
`;
